neo/Network/P2P/ProtocolHandler.cs

```diff
@@ -95,20 +93,22 @@ protected override void OnReceive(object message)
                 case "mempool":
                     OnMemPoolMessageReceived();
                     break;
+                case "ping":
+                    OnPingMessageReceived(msg.GetPayload<PingPayload>());
+                    break;
+                case "pong":
+                    OnPongMessageReceived(msg.GetPayload<PingPayload>());
```

Comment:
Is it correct @ixje @erikzhang `msg.GetPayload<PingPayload>` for Pong?

Reply:
Yes, there is only 1 payload for both. 

@erikzhang I don't know what your intention of the added `nonce` field was, but should that maybe have been included in the response as a way of confirming that the `pong` is a response to our `ping`?

e.g  
-> `ping` generate `nonce` e.g. `123`
<- `pong` with `nonce` set to the received `PingPayload.nonce` e.g. `123` in this example

Reply:
> but should that maybe have been included in the response as a way of confirming that the pong is a response to our ping?

Good idea!

Reply:
> Yes, there is only 1 payload for both.

Thanks for the clarification. So, perhaps rename to PingPongPayload, just for clarity? That really looked like a typo me :)